MANIFEST.in

```diff
@@ -5,3 +5,4 @@ recursive-include sklearn *.c *.h *.pyx *.pxd *.pxi *.tp
 recursive-include sklearn/datasets *.csv *.csv.gz *.rst *.jpg *.txt *.arff.gz *.json.gz
 include COPYING
 include README.rst
+include pyproject.toml
```

Comment:
This is needed because we need this file in the `sdist`? Otherwise in the binary releases we don't need to have it. Is my understanding correct?